### PR TITLE
Add examples for `nn.CosineEmbeddingLoss`

### PR DESCRIPTION
**Pull request by minhlong94**

### CODE REVIEW

File: torch/nn/modules/loss.py [hunk 1]
```diff
@@ -1269,6 +1269,15 @@ class CosineEmbeddingLoss(_Loss):
         - Input2: :math:`(N, D)` or :math:`(D)`, same shape as Input1.
         - Target: :math:`(N)` or :math:`()`.
         - Output: If :attr:`reduction` is ``'none'``, then :math:`(N)`, otherwise scalar.
+
+    Examples::
+
+        >>> loss = nn.CosineEmbeddingLoss()
+        >>> input1 = torch.randn(3, 5, requires_grad=True)
+        >>> input2 = torch.randn(3, 5, requires_grad=True)
+        >>> target = torch.ones(3)  # To create -1 Tensors: torch.full((3,), fill_value=-1)
```
Comment (mikaylagawarecki) [inline torch/nn/modules/loss.py:1278]: nit: leave out the comment here

**Review by mikaylagawarecki — COMMENTED** [review]
Looks good other than one nit.

To fix the failing lint job running `make setup_lint` and `make quicklint` from your local pytorch directory and following the corresponding instructions should be sufficient, see [here](https://github.com/pytorch/pytorch/blob/4721553431e968ee5e90753346d92252cce98e84/Makefile#L25-L34)

**Comment by minhlong94** [comment]
Hi @mikaylagawarecki , can you show me the instructions to do that? I am still new to the contribution process.

**Review by mikaylagawarecki — APPROVED** [review]
Looks like you figured the lint out, thanks!
